### PR TITLE
Security gets extra access on higher alert levels

### LINKED ISSUE #12078: idea that makes too much sense to not be in the game
alert levels = more sec access
green - normal sec access
blue - sec basic access to all departments
red - sec all access to all non-command departments
delta - sec all access to all departments

### PR DESCRIPTION
**Pull request by DeityLink**
While searching for issues to fix or close I came across (Fixes #12078) which indeed makes too much sense that I wonder how it's not a thing already.

The issue that suggested it got quite some support but of course I'm looking forward to your feedback below.

:cl:
* experiment: Security officers (or anyone with basic security access) now get basic department access on Code Red or Delta, allowing them to enter science, medbay, the cargo lobby or the engineering foyer. This only applies to airlocks, windoors, and access buttons. Remember to lynch sec if they abuse it.

### CODE REVIEW

**Comment by Dacendeth** [comment]
Not sure about the code blue part.

**Comment by BomberBro** [comment]
>On Code Delta, they get full access.


Seems sort of silly for a malf AI (the only way to get code delta) to automatically give every sec ID all access when going loud.

Other than that, I dislike this PR because it further brings sec into the fashie future, but the basic department access thing seems alright on code red to me.

**Comment by PrimeDSS13** [comment]
make it only basic access on code red and that's it
this is the strongest seccreep I've seen in ages

**Comment by DeityLink** [comment]
I kinda wish we'd let the feature run at full power before nerfing it, and bwoink sec to not run code blue 4noraisin. I'll wait and see how the discussion goes for a while before making any change.

**Comment by Sooose** [comment]
I really don't like the code blue portion because we all know how stupid of a reason any head can give for going code blue and no one will bat an eye. Then we've got these red goons being a pain in the ass in any department they decide to stroll into like they own the place. I think the code red portion is mostly fine just because of the requirements around going code red in the first place. Keep in mind the code red portion by itself is already a massive boon to sec in all regards and lets them mobilize to eliminate the antag like never seen before. (Take all of this with a grain of salt as I haven't played in months.)

**Comment by Kurfursten** [comment]
Code blue is too easy to escalate to 4nr and this isn't the hill to die on in terms of enforcement

**Comment by calzilla1** [comment]
Roundstart code blue? Just another step towards the /tg/ scrungularty...

**Comment by gurfan** [comment]
yes to code red, no to code blue/delta

**Comment by Blithering** [comment]
I agree, so long as blue is "reasonably suspected threat" and red is "confirmed threat" on paper it could be used for anything. Blue should ideally be "confirmed threat" while red would be  "active combat is happening"

**Comment by PrimeDSS13** [comment]
but there are confirmed threats every round

**Comment by bathosbathtime** [comment]
Don't like the idea on its face but as an experimental merge we could learn a lot from the experience so this has my support. How to balance it is beyond me but I do personally work off the philosophy that it is harder to have someone forget a negative experience than it is to improve a neutral experience.

**Comment by Jackreturnswithaveangenace** [comment]
I support experimental merges. Also the code red part seems to make more sense plus it would give people more of a reason to jump through the hoops to call code red instead of just "get the ERT over here".

**Comment by Eneocho** [comment]
Considering most of times command instantly jumps to code blue minutes after round start, this effectively means that sec gets a pseudo-permanent basic department access buff, since you so rarely go back to code green after someone jumps to code blue. This would also heavily discourage any antag attempt that isn't murderbone since they could just waltz in at any time, check on your whatever and deem you a threat. If you don't escape, they kill or brig you, if you do escape, they can jump to code red and get even more access meaning you're fucked and can't hide for shit.
And the delta part is just retarded. Why would a malf AI give the gun-toting guys access to its core. Why would they need AA instead of AI access on delta anyway? Most of the times the malf AI just sits on the AI core.

**Comment by Eneocho** [comment]
Also, if this were to be a thing, it would make sense for medstaff (or at least paramedics) to get extra access since something something combat, harm, medicine, etc.

**Comment by Dacendeth** [comment]
> Also, if this were to be a thing, it would make sense for medstaff (or at least paramedics) to get extra access since something something combat, harm, medicine, etc.

Paramedics gets basic sec access so they get these elevated accesses as well, all heads of staff too.

**Comment by SonixApache** [comment]
maybe combine it with armory locks / unlocks
i think blue and delta are unnecessary - on blue, sec can probably get the necessary access from whoever raised it to blue and on delta the whole crew is probably already helping sec break into wherever everyone needs to be

**Comment by Dacendeth** [comment]
It seems like everyone agrees sec getting access on code red is a good idea, at the very least.

**Comment by drawsstuff** [comment]
Code blue is a bit too easy to abuse, I agree with delta (...maybe) and red.

**Comment by DeityLink** [comment]
Following what seems to be the general consensus, I've changed it to basic department access on code red/delta only.

File: code/modules/security levels/keycard authentication.dm [hunk 1]
```diff
@@ -214,46 +214,25 @@ var/global/maint_all_access = 0
 	//maint access for all during radstorm, or when enabled via keycard
 	if(maint_all_access && check_access_list(list(access_maint_tunnels)))
 		return 1
-	//bonus sec access on some alert levels
-	if (security_level > 0)
+	//sec gets basic department access on Code Red or Delta
+	if (security_level > 1)
```
Comment (Kurfursten) [inline code/modules/security levels/keycard authentication.dm:218]: defines (and at line 226, 234, etc)

**Comment by Kurfursten** [comment]
> 
> 
> Following what seems to be the general consensus, I've changed it to basic department access on code red/delta only.

While good to listen to feedback, what are the odds people would actually come back into a PR and reverse their emojis? Might've been better to make it as a new PR essentially, even though that's a hassle.

**Comment by Eneocho** [comment]
Remove the delta extra access, since it makes absolutely no sense.

**Comment by SonixApache** [comment]
i don't understand why you'd think that sec should get extra access on red but then lose that access on the level above red

**Comment by ShiftyRail** [comment]
I think it's a bad idea. Codes should have a fluff effect and let people trust/coop with security more, but not give security innate advantages. This is too easy to abuse, and even if you enforce proper code escalation, it will be hard to convince officers to downgrade a code if it means losing their access.

**Comment by xeroxemnas** [comment]
powercreep

**Comment by drawsstuff** [comment]
> I think it's a bad idea. Codes should have a fluff effect and let people trust/coop with security more, but not give security innate advantages. This is too easy to abuse, and even if you enforce proper code escalation, it will be hard to convince officers to downgrade a code if it means losing their access.

Code red already gives security innate advantages you dumb dumb, in fact they almost completely change the game.

As for the abuse, since when do security officers have access to communication consoles and the authority to change the codes? They aren't heads, and just about any head can revoke all of security's access with a single button press on a comms console. 

When it comes down to it, when there's a cult in xenobiology, security is going to blow the door off the hinges anyways using hacking tools, a battering ram, windows, ect. This PR just means they can walk down the main hallway of science and not leave a gaping hole for the tide to rush through like an infection to an open wound; It's only department hallway access, even on code red everything other than the main entry is off limits. There's nothing _to_ abuse. If this PR included full department access such as say, a chemistry lab, RnD, ect, then it would be abusable, but currently, there's nothing substantial with this claim.

**Comment by Kammerjunk** [comment]
this looks like a lot of text, but that's cause there are several things to address and also things that should have been addressed. skip to the last paragraph for summary

I don't like sec constantly being treated as warranting special treatment. It's no wonder half the crew is sec officers when we encourage a culture where they're enactors of the law and therefore above it, where they're designed and intended to be prepared in case of nuclear operatives or SHODAN or The Blob (The Antag Formerly Known As An Unknown Biohazard) instead of station security.
(The second part of that is also exacerbated by the ambient normalisation of antags being present and active in every single in-universe shift through e.g. the roundstart report.)

I also have to take issue with the idea that there's nothing to abuse if they only have basic department access. reasons for this:
1. I'll say right now that sec storming into a department to take over their jobs, which they do because they don't trust that job to anyone who isn't wearing red, is extremely annoying
Is it obvious I'm chiefly talking about medbay?
2. If there's "a cult in xenobiology," they still need to blow the airlock in the little maint room to get there because it requires xenobio access. If there's a cult in virology, same story.
3. As an aside, even ignoring those 2 things, "they don't get _all_-access" just feels like a kinda weak argument for giving them extra access at all. I understand that it was said in response to an argument _against_ the PR, but then we're short an argument _for_ it.

We give things like this to sec though we wouldn't dream of giving them to anyone else, just because... what? They need to fight the antagonist? This PR specifically looks like it gives access to sec in the name of convenience, but it never actually says why sec should have more access beyond "let's see what happens."
Even with only code red, you create an environment where sec is allowed to fuck with anyone who exits their department _and_ sec is allowed to enter their departments to fuck with them. The entire station becomes an antag hunting ground.
I can't overstate how (((imbalanced))) that is for the antagonist. If we have any gamemodes that warrant sec being given powers this extreme, I'd say we should rather look at those gamemodes.

**Comment by ShiftyRail** [comment]
You cannot compare the ability to call the ERT - if you to can get your team to a comms console with a head-level access - or the ability to get combat borgs - if you have secured robotics - to a an all-purposes general access.

>When it comes down to it, when there's a cult in xenobiology, security is going to blow the door off the hinges anyways using hacking tools, a battering ram, windows, ect. This PR just means they can walk down the main hallway of science and not leave a gaping hole for the tide to rush through like an infection to an open wound;

But they should have to do all that. It's cult, you are allowed to take the gloves off and be a shitter. But you need to defend yourself and be ready to explain your reasoning to others. Alternatively, get emergency access from your local (acting) captain or HoP, which are usually present when it's cult. Simply getting a sesame to enter xenobio feels too cheap. 
I will personally defend you if you do this during cult, it's normal, it's what you're expected to do. Don't cheapen quick decision-making and using your skills to enter areas with this PR.

>There's nothing to abuse.

Disagree, a resourceful man can exploit this small access a lot. For comparison, this gives code red officers the same access as the HoS (minus bridge and armoury). That's a lot.

**Comment by drawsstuff** [comment]
> You cannot compare the ability to call the ERT - if you to can get your team to a comms console with a head-level access - or the ability to get combat borgs - if you have secured robotics - to a an all-purposes general access.

Your coder is showing. I meant none of those things. I meant being able to field execute people for not following orders. As for the other comment, again, this PR wouldn't let you get into xenobio. I think I'll draw up a quick explanation since people seem to not understand what this PR does.

**Comment by Kammerjunk** [comment]
yes, it wouldn't let you get into xenobio. therefore, sec would still have to break the airlocks into xenobio. that breaking begins (on Box) at the aft door in the little room that you pass through when doing maintenance loops:
image

**Comment by ShiftyRail** [comment]
If it "let you field execute people", then surely it lets you hack airlocks and the point of that PR is moot.
I want code red to be about creatively finding way to assert dominance over civvies by stealing their IDs, conscripting them, etc, etc. If I am given automatically more access just because the heads pressed a funny button it's not fun.

**Comment by drawsstuff** [comment]
**Before Code Red**
scinorm


**After Code Red**
sciorm



This isn't exploitable, it's quality of life. And above all else it means I don't again, leave a gaping open wound for pillaging greyshits to flood in.

**Comment by Kammerjunk** [comment]
yes, as I said, I'm aware.
coincidentally, that little strip of green is also accessible to said pillaging greyshits

**Comment by drawsstuff** [comment]
> yes, as I said, I'm aware.
> coincidentally, that little strip of green is also accessible to said pillaging greyshits

Some folks don't seem to be. The little strip of green isnt main department access.

**Comment by ShiftyRail** [comment]
And access to these corridors let you chase people so much more efficiently, attack from different angles, always keep an eye on 80% of science and be just a window-breaking away from everything else - a window you can break from inside instead of staying in the hallway.

I do not like to use gamer words but "map control" is the name of the game here. It's also why the morgue backdoor was such an advantage to security before it got removed. 

I've played sec and HoS enough to know that access to those inner corridors make planning any coordinated assault or sweep on any department trivial. You either underestimate or underplay the advantage this gives you.

**Comment by drawsstuff** [comment]
Precisely why it's wanted. Chasing. Don't bring up window breaking, RND is just one window break from the outside as well, you know this as well as me. This PR more or less gives security greater ease of mobility in eliminating wandering threats, not so much entrenched ones.

**Comment by ShiftyRail** [comment]
Well then it's retarded, there's no fun if you don't have to work your way around enemy strongholds, and keep in mind there's more than one gamemode in ss13 - traitors, vampires, etc, all would appreciate a little privacy in their dept.

I'm saying this because I like antags taking risks and keeping people alive or revivable. I cannot support this idea and not condemn this attempt at removing a safe heaven for antags to do their gimmicks. It doesn't even include toxins or xenobio (where most of the boring round-ending stuff happens) so there's no point.

And I can't believe you'd equate breaking a window from a public hallway and breaking a window from inside the department. It's miles apart in terms of what you can do.

**Comment by Kammerjunk** [comment]
I'm not sure how to properly lay out this logic, but I'll try here:
- The crew at large getting access to basic departmental rooms when the doors get battered down is bad. It's bad because then they can pillage everything that those hallways connect to. The best way for them to pillage is through these hallways.
- Security getting access to basic departmental rooms through their IDs is good. It's good because it isn't abusable. It isn't abusable because they don't get access to anything that those hallways connect to. They cannot use these hallways to get access to other department areas any better than if they did not have access.

My questions are:
Why is it good that security gets that sort of access, but bad when everyone else does? Is catching the antagonist the only reason? If so, I brought that up in the last paragraph of my first comment.
Why does security's wish to catch the antagonist supersede the wishes of everyone else to not have security constantly running through their departments?
I don't think security deserves to automatically catch the antagonist while bypassing everything ordinarily required to _get to_ the antagonist.

**Comment by APCUnlocked** [comment]
This is bad because if this gets merged, i'll have even less excuses to use the oh-so-underutilized battering ram. Also for the reasons Shifty and Kammer have brought up.

**Comment by bathosbathtime** [comment]
If they are getting more access for emergency purposes, shouldn't engineering and medical staff also gain these access increases?

**Comment by SonixApache** [comment]
Code red being a "sit tight while the station is made safe and secure again" situation works gameplay-wise and RP-wise.
Security staff, the people in charge of ensuring the station is safe and secure, getting more access to ensure they are able to do exactly that makes sense, gameplay-wise and RP-wise.
As Shifty said, you are expected to get extra access already anyway. Be it in the form of hacking tools or actual legitimate ID card changes (which then need to be individually manually removed at a later point).
Two heads of staff have already discussed and coordinated to engage code red, do they really need to spend more time fiddling with ID cards when the act of announcing code red could do the same thing?

Having an issue with having sec in your department is fine gameplay-wise and RP-wise. Armored meatheads shoving you around and preventing you from doing the things you want to do is annoying, but this can lead to interesting things during and after code red.
Having sec take over your work however is not fine. But that's a policy issue that we can't solve through code.

Shifty's notion of "Security should work together with the general crew / the general crew should work together with security on code red" is interesting but completely out of place and in the realm of fantasy when it comes to our playerbase/server.
I do agree with Kammer however in that giving them general access to get around isn't that useful. They should be given department-specific access as well, so they don't have to go around ramming down xenobio.

**Comment by DeityLink** [comment]
> 
> 
> > Following what seems to be the general consensus, I've changed it to basic department access on code red/delta only.
> 
> While good to listen to feedback, what are the odds people would actually come back into a PR and reverse their emojis? Might've been better to make it as a new PR essentially, even though that's a hassle.

This PR doubles as a place for discussion, I'd like as many people as possible to reach a consensus first.

**Comment by DeityLink** [comment]
> 
> **Before Code Red**
> 
> **After Code Red**
> 
> This isn't exploitable, it's quality of life. And above all else it means I don't again, leave a gaping open wound for pillaging greyshits to flood in.

@drawsstuff I thought I'd mention that there's also the Borer Pen, and when I tested I believe that Telesci uses general science access. Only Robotics, Xenobio, R&D and the Outpost Shuttle aren't included (and the RD's office ofc)

**Comment by Kammerjunk** [comment]
It doesn't make any sense gameplay-wise, even if you say that it does. It's a benefit for Antaghunter Securityman, but it's to the detriment of everyone else - not least of all the antag he's hunting. Distrustful sec officers barging in and doing another department's job to ensure they don't put their faith in an antag and thus Lose the Game is not a policy issue we've solved in 8 years, and making it worse with this PR is a poor step.

\>do they really need to spend more time fiddling with ID cards when the act of announcing code red could do the same thing?
_Yes_. Of course they do. Having to go through a couple of extra steps in response to an individual threat (and they are by nature individual and hard to blanket cover) is not the end of the world, not the round. Automating it is lifeless, end-all, and absolutely abusable by all the same arguments that deny access to everyone else. Far better and more creational of "situations" is the current state, where a real person makes a decision that they can later be held accountable for.
Security does not have the god-given right to catch the antagonist.

On the bright side, Sonix accelerates the idea before the PR has even been merged. I did consider saying in my original comment something to the effect of "what's next? Does giving security full department access make too much sense too?" I chose not to because it felt snide towards the arguments made so far, and because it might prevent someone who held that opinion from giving it.

@DeityLink yup, telesci only requires general sci access. I've used that a few times as geneticist, who gets that access.
(IIRC there's a specific Telescience access, but the airlocks only require any instead of all.)

**Comment by DeityLink** [comment]
**Also here's another idea I had which might not be much harder to implement, and would give the HoP a new job:**

A new computer console that can manage "exceptional permissions".

For example let's say that Virology got wrecked by meteors, this would allow the HoP to grant Medbay/Virology access to people with Engineering access for the next 20 minutes so they could repair it. This exceptional access could be set to automatically revert after a set time, or until toggled off. Alternatively exceptional access could be set to specific ID numbers, essentially allowing the HoP to control access at range (although the console could only grant access, not remove it).

I think the battering ram is still useful as a quick solution against welded/bolted doors.

**Comment by Apogee-dev** [comment]
> **Also here's another idea I had which might not be much harder to implement, and would give the HoP a new job:**
> 
> A new computer console that can manage "exceptional permissions".
> 
> For example let's say that Virology got wrecked by meteors, this would allow the HoP to grant Medbay/Virology access to people with Engineering access for the next 20 minutes so they could repair it. This exceptional access could be set to automatically revert after a set time, or until toggled off. Alternatively exceptional access could be set to specific ID numbers, essentially allowing the HoP to control access at range (although the console could only grant access, not remove it).
> 
> I think the battering ram is still useful as a quick solution against welded/bolted doors.

"exceptional permissions" sounds similar to the temporary passes some other codebases have without needing to individually distribute passes, and I'm all in favor of it. Could be nice to have department-limited versions of that console in each of the head offices, perhaps

**Comment by Kurfursten** [comment]
I very rarely agree with kammerjammer so when we do agree she's definitely right. There's a big question mark over why for this PR.

This *almost* makes TG's "why it's good for the game" mandatory description a good idea. Almost.

**Comment by SonixApache** [comment]
>it's to the detriment of everyone else

I don't see how it's to the detriment to everyone else for sec to do their job. Especially since they will be doing the exact same thing regardless, and in harder to reverse ways otherwise - except with a slight time delay.
>Distrustful sec officers barging in and doing another department's job to ensure they don't put their faith in an antag and thus Lose the Game is not a policy issue we've solved in 8 years, and making it worse with this PR is a poor step.

This is still not a code issue but a policy issue. If you personally have issues with sec (or others) barging in taking over workstations, make it policy that they can't or are at least not supposed to. I don't feel this is strongly relevant to the discussion at hand.

>Yes. Of course they do. Having to go through a couple of extra steps in response to an individual threat (and they are by nature individual and hard to blanket cover) is not the end of the world, not the round. Automating it is lifeless, end-all, and absolutely abusable by all the same arguments that deny access to everyone else. Far better and more creational of "situations" is the current state, where a real person makes a decision that they can later be held accountable for.

Those were a lot of words that amounted to absolutely nothing. In my experience, when extra access is required, cards are just blanket-upgraded to ensure airlock access won't be the deciding factor in who comes out on top. Requiring two real persons to press one button simultaneously is not so different from having one person touch a few card for a few seconds that it deserves some sort of merit.
>Security does not have the god-given right to catch the antagonist.

medical does not have the god-given right to clone people
engineering does not have the god-given right to set up the supermatter
cargo does not have the god-given right to ship crates
is catching antagonists not just an extension of enforcing the law and protecting the crew; the reason security exists?

**Comment by Kammerjunk** [comment]
\>medical does not have the god-given right to clone people
\>engineering does not have the god-given right to set up the supermatter
\>cargo does not have the god-given right to ship crates
yup, and we don't codewise automate synthmeat production if there are many bodies in medbay, or plasma cooling if the SMESs are out of power.

It is notably different. We would be kidding ourselves and nobody else to suggest that this PR would make (relevant) heads less hesitant about increasing the alert level. That means the alert level would still get raised for the same justifications. That means sec would get these accesses because the alert level got raised, not because an HoP or (acting) captain decided they needed that access. Therefore, there is a lesser accountability for sec roaming around in places where they really have no legitimate business being, and no greater authority letting them in there.

It's to the detriment of everyone else because everyone else ends up with sec rummaging through their departments, and because the entire intent of the PR is to let them catch the antag wherever it goes.
It would be retarded to look only at "what does department want" and try to make that easier. I hope I don't need to explain why we need to broaden that scope until it instead becomes "...and does it make the game better for everyone?" This is certainly a convenience for sec, but sec _does not exist to catch the antagonist_ and does not deserve to be given ever-better tools to do so more efficiently. Medbay doesn't deserve to have people exit the medbay in a healthy state. Engineering does not deserve to have power flowing out. These are things that the departments accomplish: medbay fills the fridge at roundstart and makes more pills as needed, engineering cools the plasma, security goes through the legal hoops instead of playing soldier boys, and these all affect the station and the round at large.
Sec doing this is especially an issue relating to this PR for these reasons: they will be doing it more often, and there's less to rein them in.

Catching the antagonist is a subset of why security exists. It's relevant when there is an Enemy of the Corporation. It is not why security exists, and therefore not what security should be designed around, either thematically, by administration, or by features like this.

**Comment by KillFurries** [comment]
The awful arguments from the support side just show what kind of manchildren you're dealing with here. The position is that security can do no wrong and they're the good guys who keep everyone safe, so obviously they should get all the access they need. You just need to ''sit tight'' and let them handle it. These freaks have no regard for the rights of non-security players. The security players are worse than the antags.
There are systems and rules in place to keep security players in check so they don't abuse their power, don't grief and people aren't forced to interact with them. Security isn't ''the people in charge of ensuring the station is safe and secure'' as Sonichu says. Security is staffed by other players who more often than not are complete dickheads. This tone deaf PR comes just months after there were several PRs to keep security players from getting into medbay through the morgue.
These people deserve to be perma-locked in a post 9/11 post covid wagie cage getting ''protected'' by their leaders and police that can never do no wrong who get to violate all your rights, go wherever they want whenever they want. People who supported this PR should have it tattooed on their static's face and they should be rounded up every round and put in little safe boxes to keep the jobs of our great security officers easier.

> “Those who would give up essential liberty to purchase a little temporary safety, deserve neither liberty nor safety.” - Shrek

**Comment by DeityLink** [comment]
> I very rarely agree with kammerjammer so when we do agree she's definitely right.

Gotta say I feel the same.

>There's a big question mark over why for this PR.

The "why" was #12078 which regardless of my opinion seemed like a feature that people wanted to see, at least back in 2016. It was just a dozen lines of code so by actually writing those we now we actually got a discussion with both sides.

Having read Kammer's points, "Automating it is lifeless, end-all" resonated with me considering that's the main reason why I removed boxing from cult conversions. The more I think about it, the more the new HoP console I envisioned feels like it would be better than the automated system suggested by ghost back in 2016.

Unless I get some rock solid argument in favor of this PR I'll close it at the end of the day, along with the issue that suggested it 5 years ago.

**Comment by drawsstuff** [comment]
After rethinking this somewhat in my gym time, I think I'll withdrawal my support for the time being. The idea is rather nice on paper, but there's a few things that sit wrong with me now that I think about it.

1. It decentivizes the HoS from rounding up his officers and opening said doors himself.
2. Battering ram neglect
3. It doesn't make sense for only security to gain additional access when the code red could be more of a medical or engineering emergency in nature.

With that being said, my two cents for those who like to see this in might find it easier to garner support in a different form. Perhaps additional access can be granted per department by swiping two head cards similar to code red? Variety is the spice of life after all.

**Comment by SugarBone** [comment]
> Code red already gives security innate advantages you dumb dumb, in fact they almost completely change the game.
> 
> As for the abuse, since when do security officers have access to communication consoles and the authority to change the codes? They aren't heads, and just about any head can revoke all of security's access with a single button press on a comms console.
> 
> When it comes down to it, when there's a cult in xenobiology, security is going to blow the door off the hinges anyways using hacking tools, a battering ram, windows, ect. This PR just means they can walk down the main hallway of science and not leave a gaping hole for the tide to rush through like an infection to an open wound; It's only department hallway access, even on code red everything other than the main entry is off limits. There's nothing _to_ abuse. If this PR included full department access such as say, a chemistry lab, RnD, ect, then it would be abusable, but currently, there's nothing substantial with this claim.

Uh, so you are saying that we should remove security gimmicks (like door breaching and using the battering ram) just so you could frag faster? That seems like blatant powergaming to me, which only serves to dumb down the overall process. As a person that occasionally plays sec, I wouldn't want to remove these gimmicks, as they add to the overall experience and force you to interact with other players (AI, people in the dept and etc.). Overall, it is a shitty PR made by equally shitty players. It should be closed without any discussion.

**Comment by DeityLink** [comment]
> Unless I get some rock solid argument in favor of this PR I'll close it at the end of the day, along with the issue that suggested it 5 years ago.

Well that's it then

**Pull request closed without merging**